public/waitingroom.html

```
@@ -0,0 +1,163 @@
+    <a href="#">LEARN ▾</a>
+    <a href="/courses.html">COURSES</a>
+    <a href="#">COMMUNITY ▾</a>
+    <a href="#">RESOURCES ▾</a>
+    <a href="#">ABOUT ▾</a>
+  </div>
+  <div class="nav-search">
+    <svg width="13" height="13" fill="none" stroke="currentColor" stroke-width="2.2" viewBox="0 0 24 24"><circle cx="11" cy="11" r="8"/><path d="m21 21-4.35-4.35"/></svg>
+    <input type="text" placeholder="What do you want to learn?"/>
+  </div>
+  <div class="nav-icons">
+    <button class="icon-btn">💬</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)